### ISSUE TITLE
Error when running analyze/morris.py on a set of data that has zero sensitivity

### ISSUE THREAD

**Issue by buck06191** [SALib/SALib#153]
Hi,
I've recently inherited a project in my research group and we use SALib in our software, BCMD [repo here](https://github.com/bcmd/BCMD) 
A user of our software has been running sensitivity analysis on our model using their measured data as driving inputs. A number of parameters have a sensitivity of zero - the reasoning behind this needs more investigation but it could potentially be reasonable.
The problem is that the following error is thrown

```python
> Traceback (most recent call last):
  File "postproc.py", line 239, in <module>
    postproc(dir)
  File "postproc.py", line 54, in postproc
    calcDistances(dir, config)
  File "postproc.py", line 207, in calcDistances
    grid_jump=config['info']['jump'])
  File "/home/buck06191/anaconda3/envs/bcmd/lib/python2.7/site-packages/SALib/analyze/morris.py", line 83, in analyze
    ee = compute_elementary_effects(X, Y, int(Y.size / num_trajectories), delta)
  File "/home/buck06191/anaconda3/envs/bcmd/lib/python2.7/site-packages/SALib/analyze/morris.py", line 233, in compute_elementary_effects
    np.divide(ee, delta, out=ee)
TypeError: ufunc 'divide' output (typecode 'd') could not be coerced to provided output parameter (typecode 'l') according to the casting rule ''same_kind''
```

I've looked into the numpy function and have worked out that when ee is all zeros, all integers, and delta is a float, it's unable then pass the resulting array directly back into ee. 

Could this be fixed by ensuring that either 

1. ee and delta are always of the same type OR
2. ee is converted into floats before dividing.

This is using Python 2.7
Thanks

**Comment by buck06191** [comment]
Apologies, I've fixed the issue from our end. Our code was sending an array of integers to the function. Have now fixed it by ensuring we don't send integers.

**Comment by willu47** [comment]
Thanks for the bug report @buck06191.  I think an error should be raised in SALib if the inputs are not floats.  That would make this problem easier to identify.

**Comment by willu47** [comment]
Closed by #154
